src/KeyboardLayouts/ImprovedKeylayouts.h

```diff
@@ -24,11 +24,59 @@ THE SOFTWARE.
 // Include guard
 #pragma once
 
-#if !defined(LAYOUT_US_ENGLISH)
-#error This API does not support non US english layouts.
+// Keyboard Modifiers
+enum KeyboardMods : uint16_t {
+	MOD_LEFT_CTRL		= 0b0000000100000000,
```

Comment:
Using (1<<8) etc would be better to me.

Reply:
Yeah, me too. But (1>>15) throws a compile error complaining the resulting number does not fit into uint16_t. It was late, so I went with the lazy solution...

Reply:
I guess you mean 1<<15 not 1>>15? Maybe you need to add a cast?

Reply:
Yeah, cast was needed. Fixed.